### ISSUE TITLE
Focus management consideration

### ISSUE THREAD

**Issue by adamsilver** [alphagov/accessible-typeahead#76]
Currently the component uses both `.focus()` *and* `aria-activedescendant` to announce the focused option.

According to spec a composite widget like this, should keep focus on the textbox and focus should be managed by aria-attributes (and CSS for visual highlighting), which is what I have done in my [demo](http://adamsilver.github.io/kitty/src/demos/Autocomplete/).

It would be good to investigate how well this works for AT.

**Comment by tvararu** [comment]
Agree this would be good to investigate.

Historical context: While spiking initial implementations for this I attempted to use `aria-activedescendant` on its own incorrectly, leading me to falling back to the current `.focus()` approach, which I saw work well in [Leonie's prototype](http://ljwatson.github.io/design-patterns/autocomplete/index.html).

**Comment by edwardhorsford** [comment]
I think the current approach seems to work well (for accessibility) - but has the downside that without focus on the input, it's not obvious that you can continue to type in to it.

**Comment by adamsilver** [comment]
Exactly that @edwardhorsford

**Comment by adamsilver** [comment]
aria-activedescendant only works in Voiceover if focus is on the parent (with the attribute).

Therefore I suggest you leave the implementation as-is, because the input element cannot be a parent of the suggestion menu.

Would suggest this issue is closed.

**Comment by tvararu** [comment]
@adamsilver thanks for investigating further! That's an annoying behaviour. If that's the case then yes we can close this for now.
